src/vorta/store/settings.py

```diff
@@ -48,8 +48,11 @@ def get_misc_settings() -> List[Dict[str, str]]:
             'value': True,
             'type': 'checkbox',
             'group': startup,
-            'label': trans_late('settings', 'Open main window on startup'),
-            'tooltip': trans_late('settings', 'Open main window when the application is launched'),
+            'label': trans_late('settings', 'Show main window of Vorta on startup'),
```

Comment:
To me `startup` makes this setting sound like the autostart setting. Can you find a different word that describes the same?

Reply:
@real-yfprojects I've committed some changes. See how it looks now.